q2_fondue/plugin_setup.py

```diff
@@ -57,7 +58,8 @@
     'paired_reads': 'Artifact containing paired-end fastq.gz files '
                     'for all the requested IDs.',
     'failed_runs': 'List of all run IDs for which fetching {} failed, '
-                   'with their corresponding error messages.'
+                   'with their corresponding error messages.',
+    'ids': 'Artifact containing retrieved SRA accession IDs.'
```

Comment:
Since this action fetches run accession IDs how about specifying the description accordingly?